hydra-cluster/config/devnet/genesis-conway.json

```diff
@@ -1,3 +1,38 @@
 {
-  "genDelegs": {}
+  "genDelegs": {},
```

Comment:
This is actually not part of the genesis-conway.json on mainnet anymore: https://book.world.dev.cardano.org/environments/mainnet/conway-genesis.json

Could: remove this.